### PR TITLE
created jdownloader profile

### PR DESCRIPTION
**Pull request by veloute**

### CODE REVIEW

File: etc/JDownloader.profile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Firejail profile for JDownloader
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/JDownloader.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# generic gui profile
```
Comment (SkewedZeppelin) [inline etc/JDownloader.profile]: remove generic comment

File: etc/JDownloader.profile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+noroot
+nosound
+notv
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+
+disable-mnt
+# private
```
Comment (SkewedZeppelin) [inline etc/JDownloader.profile]: remove commented lines

File: etc/JDownloader.profile [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Firejail profile for JDownloader
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/JDownloader.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# generic gui profile
+# depending on your usage, you can enable some of the commands below:
+
```
Comment (SkewedZeppelin) [inline etc/JDownloader.profile]: is the following not needed?

```
# Allow access to java
noblacklist ${PATH}/java
noblacklist /usr/lib/java
noblacklist /etc/java
noblacklist /usr/share/java
```

File: etc/JDownloader.profile [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+include /etc/firejail/disable-common.inc
+# include /etc/firejail/disable-devel.inc
```
Comment (SkewedZeppelin) [inline etc/JDownloader.profile]: any specific reason disable-devel.inc is commented?

File: etc/JDownloader.profile (hunk 1 above)
Comment (veloute) [reply etc/JDownloader.profile]: forgot about that. done.

File: etc/JDownloader.profile (hunk 4 above)
Comment (veloute) [reply etc/JDownloader.profile]: your suggestion for adding the noblacklist options for java fixed the issues i had with that - uncommented.

File: etc/JDownloader.profile (hunk 3 above)
Comment (veloute) [reply etc/JDownloader.profile]: thanks. done.

File: etc/JDownloader.profile (hunk 2 above)
Comment (veloute) [reply etc/JDownloader.profile]: done.

File: etc/JDownloader.profile [hunk 5]
```diff
@@ -5,16 +5,25 @@ include /etc/firejail/JDownloader.local
...
 include /etc/firejail/disable-common.inc
-# include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-devel.inc
 include /etc/firejail/disable-interpreters.inc
 include /etc/firejail/disable-passwdmgr.inc
 include /etc/firejail/disable-programs.inc
 include /etc/firejail/disable-xdg.inc
 
+mkdir ${HOME}/.jd
+whitelist ${HOME}/.jd
```
Comment (SkewedZeppelin) [inline etc/JDownloader.profile]: Add
```
whitelist ${DOWNLOADS}
include /etc/firejail/whitelist-var-common.inc
```

File: etc/JDownloader.profile [hunk 6]
```diff
@@ -32,14 +41,9 @@ seccomp
 shell none
 
 disable-mnt
```
Comment (SkewedZeppelin) [inline etc/JDownloader.profile]: should other drives be disabled? if this is a generic downloader people might want download on another drive

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**